installer/src/panels/com/jboss/devstudio/core/installer/InstallAdditionalRuntimesPanelAutomationHelper.java

```diff
@@ -32,11 +39,46 @@ public void makeXMLData(AutomatedInstallData installData, IXMLElement panelRoot)
 		panelRoot.addChild(locations);
 	}
 
+	/**
+	 * Establish the runtime_locations.properties file based on the JBoss Application Server location
+     * (if that panel is active) or minimally just the 'runtimes' directory.  This file is read at startup
+     * and affects the runtime UI preferences.  Also set the install data runtime locations variable.
+	 */
 	public void runAutomated(AutomatedInstallData idata, IXMLElement panelRoot) throws InstallerException {
 		IXMLElement locations = panelRoot.getFirstChildNamed(LOCATIONS_NODE_NAME);
 		if (locations != null && locations.getContent() != null) {
 			idata.setVariable(Unpacker.INSTALL_RT_LOCATIONS_VAR, locations.getContent());
 		}
-	}
+    	String installPath = idata.getVariable("INSTALL_PATH");
```

Comment:
I am sorry, my suggestion was wrong. I thought that we were not saving configuration specific to additional runtimes, but if runtime_locations.properties is not getting saved it means the problem is in CreateLinkPane.java and CreateLinkPanelAutomationHelper.java. The added code is not needed here. Instead method that saves runtime_locations.properties should be moved from CreateLinkPanel.java to its AutomationHelper as static method, so both classes CreateLinkPane.java and CreateLinkPanelAutomationHelper.java can use it to save configuration into runtime_locations.properties runtime detection plugin.